### PR TITLE
ADOP-1204

### PR DESCRIPTION
**Pull request by ShakiraKP**
Add additional pact testing in adoption-web

[### JIRA link](https://tools.hmcts.net/jira/browse/ADOP-1204)

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by pankajhmcts — CHANGES_REQUESTED** [review]
Once build issure resolved, please let us know as soon as possible, will merge this PR
@ShakiraKP @mohitvijay45 @rajatkgupta29 @nraghu32 @harshniavadhannla

File: charts/adoption-web/values.preview.template.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ nodejs:
   environment:
     APPINSIGHTS_KEY: '00000000-0000-0000-0000-000000000000'
     PCQ_ENABLED: 'true'
-    CCD_URL: 'http://ccd-data-store-api-adoption-cos-api-pr-249.service.core-compute-preview.internal'
+    CCD_URL: 'http://adoption-cos-api-pr-249-ccd-data-store-api'
```
Comment (pankajhmcts) [inline charts/adoption-web/values.preview.template.yaml:16]: Remove this file chane, sync with R2master

File: config/default.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ services:
     systemPassword: 'dummy_password'
     userRole: 'adoption-citizen-user'
   case:
-    url: 'http://ccd-data-store-api-adoption-cos-api-pr-249.service.core-compute-preview.internal'
+    url: 'https://ccd-data-store-api-adoption-cos-api-pr-249.service.core-compute-preview.internal'
```
Comment (pankajhmcts) [inline config/default.yaml:19]: Remove this file chane, sync with R2master

File: package.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "prepare": "husky install",
     "start": "NODE_ENV=production ts-node --transpile-only -r tsconfig-paths/register src/main/server.ts",
-    "start:dev": "NODE_ENV=development yarn start:watch",
+    "start:dev": "NODE_TLS_REJECT_UNAUTHORIZED=0 NODE_ENV=development yarn start:watch",
```
Comment (pankajhmcts) [inline package.json:11]: Remove this file chane, sync with R2master

File: src/main/steps/application/task-list/content.test.ts [hunk 4]
```diff
@@ -138,53 +138,6 @@ describe('task-list > content', () => {
             },
           ],
```
Comment (pankajhmcts) [inline src/main/steps/application/task-list/content.test.ts:139]: Remove this file chane, sync with R2master

File: src/main/steps/application/task-list/content.ts [hunk 5]
```diff
@@ -9,17 +9,10 @@ import {
   getAdoptionCertificateDetailsStatus,
   getApplicationStatus,
   getApplyingWithStatus,
-  getBirthFatherDetailsStatus,
-  getBirthMotherDetailsStatus,
-  getChildrenBirthCertificateStatus,
-  getChildrenPlacementOrderStatus,
   getContactDetailsStatus,
```
Comment (pankajhmcts) [inline src/main/steps/application/task-list/content.ts:12]: Remove this file chane, sync with R2master

File: src/main/steps/review-pay-submit/check-your-answers/content.ts [hunk 6]
```diff
@@ -130,17 +124,10 @@ const en = (content: CommonContent) => {
       childrenSummaryList(enContent, userCase),
```
Comment (pankajhmcts) [inline src/main/steps/review-pay-submit/check-your-answers/content.ts:124]: Remove this file change, sync with R2master

File: src/main/steps/review-pay-submit/check-your-answers/utils.ts [hunk 7]
```diff
@@ -361,16 +361,6 @@ export const childrenSummaryList = (
           value: userCase.childrenFirstName + ' ' + userCase.childrenLastName,
           changeUrl: Urls.CHILDREN_FULL_NAME,
         },
```
Comment (pankajhmcts) [inline src/main/steps/review-pay-submit/check-your-answers/utils.ts:363]: Remove this file change, sync with R2master

File: src/test/pact/ccd-data-store-api.test.ts [hunk 8]
```diff
@@ -4,6 +4,7 @@
 //   getServiceAuthToken: jest.fn(() => 'mock-service-auth-token'),
 // }));
 
+// // import axios from 'axios';
```
Comment (pankajhmcts) [inline src/test/pact/ccd-data-store-api.test.ts:7]: Why this file have change?

**Review by pankajhmcts — CHANGES_REQUESTED** [review]
Please make sure your branch must have PACT changes only.
@ShakiraKP

File: charts/adoption-web/values.preview.template.yaml (hunk 1 above)
Comment (ShakiraKP) [reply charts/adoption-web/values.preview.template.yaml]: these changes are from me syncing with r2master

File: src/main/steps/application/task-list/content.ts (hunk 5 above)
Comment (ShakiraKP) [reply src/main/steps/application/task-list/content.ts]: these changes are from me syncing with r2master

**Review by pankajhmcts — CHANGES_REQUESTED** [review]
I don't see jenkins file change in this PR, please enable it.

File: config/default.yaml (hunk 2 above)
Comment (ShakiraKP) [reply config/default.yaml]: > I don't see jenkins file change in this PR, please enable it.

The ccd pact test cases are not fixed so if i enable this change the PR will fail.

**Review by pankajhmcts — APPROVED** [review]

**Comment by rajatkgupta29** [comment]
https://pact-broker.platform.hmcts.net
Please verify all the Pacts are executed and green at the broker end

**Pull request closed without merging**